README.md

```diff
@@ -1 +1,228 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- **[OPTIONAL]** kindly remove the Microverse logo or replace it with your own for a professional README file. It is a general requirement for all projects, in the future if the README is incomplete or unprofessional your project will be invalidated ❌